### ISSUE TITLE
Feature request - swipe between players

### ISSUE THREAD

**Issue by eremem** [kaaholst/android-squeezer#708]
When in player view you can switch to the other players by using the combo box in the upper left corner.
It seems to me that it would improve the UX and the ease of switching between players if we could also swipe left/right to the previous/next player on this list. It would be useful not only in the "main" player view but also in the detail and playlist views.

**Comment by bvrulez** [comment]
This view displays a list of single menu items. Implementing it for a single item would be an unintuitive behaviour, because the items are small and because such a swipe should swipe just one item, not the whole screen. 

However, if the list of items is short enough to leave some space at the bottom there would be an area available for such a listener.

**Comment by eremem** [comment]
Maybe I made myself not clear enough. I'm suggesting adding swiping to the items' list in the combo box. I meant the whole actual player views I mentioned.

**Comment by bvrulez** [comment]
Yes. With single items I meant the menus (myMusic, Radio, Search, Extras, Applications, Power off, Favorites, Settings) in the basic menu you get for one certain player when you start the app (after connecting).

**Comment by eremem** [comment]
I'm still not sure if we are on the same page though:D.
Just to be sure, let's assume we have 3 players. You choose player 2 from the combo box and see the view you described. Now you swipe from the left edge and switch to the same view but for player 1. Further swiping to the left is not possible, since we reached the top of the players' list (as seen in the combo box). Now, if you swipe from the right edge of the screen, you can switch to the current view for player 2 and so on.
This behavior could be "replicated" in the player's view for the current song/stream and the player's playlist view.

**Comment by bvrulez** [comment]
yes, we are definitely on the same page. :) but thanks for clarifying. since this view contains a list of items, i am unsure if a swipe can be implemented in an intuitive way (as described) . but i will try to discuss it.

**Comment by bvrulez** [comment]
sure!
